### ISSUE TITLE
Documentação(Especificação)

### ISSUE THREAD

**Issue by victor95pc** [mainosistemas/open-nfe#26]
Precisamos de um descrição clara do que precisa ser feito, pode ser criando um board no Trello ou fazendo uma lista aqui no github, está lista pode ser feita pelo @pedropazello  ou @neilvilela, que domina a regra de negocio.

**Comment by pedropazello** [comment]
@victor95pc 
Se eu te falasse eu não sei quase nada de venda de produtos e tributos?
Por exemplo, explicar o que significa IPI é desnecessário, o máximo que pode ser feito é apontar em qual lugar da documentação da NF-e se encontra determinada informação.
Exemplo, precisa utilizar o tributo x na situação y. É uma regra de negócio da NF-e, e alguns estados possuem regras diferentes.
Acho que o máximo que dá pra fazer é explicar como cada funcionalidade _do sistema_ funciona, como por exemplo cadastro de clientes, permissões, transmissão da NF-e, como instalar o sistema, etc.
Acho que o ideal é a pessoa que fez a maior parte da funcionalidade escrever como ela funciona.
Exemplo: Desenvolvi a parte de permissões de usuários, ai eu vou e faço uma documentação dessa parte.
O que você acha?

**Comment by victor95pc** [comment]
Sim é uma boa, mas tem que cadastrar essas tarefas até agora parece um limbo, nem sei por onde começar.

**Comment by pedropazello** [comment]
Sim, concordo, tem que cadastrar as tarefas o mais rápido possível haha

**Comment by pedropazello** [comment]
Por enquanto estou criando as tarefas no repositório mesmo, depois podemos ver o melhor lugar para criar as novas.
